config/locales/en/settings.yml

```diff
@@ -125,6 +125,9 @@ en:
       valuation_comment_notification: "Valuation comment notification"
       valuation_comment_notification_description: "Send an email to all associated users except valuation commenter to budget investment when a new valuation comment is created"
       graphql_api: "GraphQL API"
+      dashboard:
+        notification_emails: "Dashboard notification emails"
+        notification_emails_description: "Enable the sending dashboard notification emails to proposal’s authors"
```

Comment:
’"Enable the sending" sounds strange to me; "Enable sending" sounds better, although I might be wrong since I'm not a native speaker.

By the way, I think we use `'` instead of `’` in English (so it would be "proposal's" instead of "proposal’s") :thinking:.